oracle/types/provider.go

```diff
@@ -13,24 +13,24 @@ type (
 	ProviderTicker interface {
 		fmt.Stringer
 
-		// Provider returns the provider for the ticker.
-		Provider() string
-		// OffChainTicker returns the off-chain representation for the ticker.
-		OffChainTicker() string
-		// Decimals returns the number of decimals for the ticker.
-		Decimals() uint64
-		// JSON returns additional JSON data for the ticker.
-		JSON() string
+		// GetProvider returns the provider for the ticker.
+		GetProvider() string
+		// GetOffChainTicker returns the off-chain representation for the ticker.
+		GetOffChainTicker() string
+		// GetDecimals returns the number of decimals for the ticker.
+		GetDecimals() uint64
+		// GetJSON returns additional JSON data for the ticker.
+		GetJSON() string
 	}
 
 	// DefaultProviderTicker is a basic implementation of the ProviderTicker interface.
 	// Provider's that utilize this implementation should be able to easily configure
 	// custom json data for their tickers.
 	DefaultProviderTicker struct {
-		provider string
-		offChain string
-		decimals uint64
-		json     string
+		Name           string
+		OffChainTicker string
+		Decimals       uint64
+		JSON           string
```

Comment:
Doing this so that all of our provider market configs can be backwards compatible.